src/System.Management.Automation/engine/CommandCompletion/ExtensibleCompletion.cs

```diff
@@ -207,4 +207,168 @@ public IEnumerable<CompletionResult> CompleteArgument(string commandName, string
             }
         }
     }
+
+
+    /// <summary>
+    ///     Base class for writing custom Argument Completers
```

Comment:
Usually we use one space after `///`. And please add final dot in public comments.